src/store/modules/accounts/actions.js

```diff
@@ -1,6 +1,6 @@
 import isEqual from 'lodash/isEqual'
 import isEmpty from 'lodash/isEmpty'
-import ae from '../../aeppsdk'
+import EpochChain from '@aeternity/aepp-sdk/es/chain/epoch'
```

Comment:
please import as documented in the [README.md](https://github.com/aeternity/aepp-sdk-js/blob/develop/README.md): `import Aepp from '@aeternity/aepp-sdk/es/ae/aepp'`